### DISCUSSION TITLE
Is there an overview of the current set of queries?

### DISCUSSION THREAD

**Discussion by mutasDev** [github/codeql#10787] [Q&A]
Does a list / an overview of the currently included queries, e.g. by their CWE / topic, exist or is there any way to check, which queries are a part of a specific query set ?

**Comment by jketema** [comment]
Thanks for your question.

For an overview of covered CWEs see https://codeql.github.com/codeql-query-help/codeql-cwe-coverage/

To tell which queries are part of a particular suite, we can use `codeql resolve queries`. For example, say we want to know which queries are part of the C++ security and quality suite, then we ensure we've downloaded the `codeql/cpp-queries` pack and run the aforementioned command:
```
codeql pack download codeql/cpp-queries
codeql resolve queries codeql/cpp-queries:codeql-suites/cpp-security-and-quality.qls
```
For query pack names and suite names also see the tip just below https://codeql.github.com/docs/codeql-cli/analyzing-databases-with-the-codeql-cli/#example-query-specifiers
